server/fly.io/src/main.rs

```diff
@@ -0,0 +1,79 @@
+use std::collections::BTreeMap;
+use std::net::SocketAddr;
+
+use axum::{
+    extract::Query,
+    routing::{get, post},
+    Router,
+};
+use influxrs::{InfluxClient, Measurement};
+
+fn main() {
+    let rt = tokio::runtime::Runtime::new().unwrap();
+    let task = rt.spawn(async move {
+        let app = Router::new()
+            .route("/", get(root))
+            .route("/record-install", get(record_install))
+            .route("/record-install", post(record_install));
+
+        // ipv6 + ipv6 any addr
+        let addr = SocketAddr::from(([0, 0, 0, 0, 0, 0, 0, 0], 8080));
+        axum::Server::bind(&addr)
+            .serve(app.into_make_service())
+            .await
+            .unwrap();
+    });
+    rt.block_on(task).unwrap();
+}
+
+async fn root() -> &'static str {
+    "Hello, World!"
+}
+
+fn get_env(key: &str) -> String {
+    std::env::var(key).expect(&format!("{key} must be set"))
+}
+
+async fn record_install(Query(params): Query<BTreeMap<String, String>>) -> String {
+    println!("Hi there {params:?}");
+
+    // FIXME: make this in main and pass it down or something?
+    let url = get_env("INFLUX_URL");
+    let token = get_env("INFLUX_TOKEN");
+    let org = get_env("INFLUX_ORG");
+    let bucket = get_env("INFLUX_BUCKET");
+    let client = InfluxClient::builder(url, token, org).build().unwrap();
```

Comment:
With [`std::sync::LazyLock`](https://doc.rust-lang.org/std/sync/struct.LazyLock.html) stablised, we could simply make it global variable:

```suggestion
static INFLUX_CLIENT: LazyLock<InfluxClient> = LazyLock::new(|| {
    let url = get_env("INFLUX_URL");
    let token = get_env("INFLUX_TOKEN");
    let org = get_env("INFLUX_ORG");
    let bucket = get_env("INFLUX_BUCKET");
    InfluxClient::builder(url, token, org).build().unwrap()
});
```